jenkins-integration/beaker/gatling/watch_puppetserver_mem.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+
+# This script outputs the memory usage of puppet server every $delay seconds
+# Example usage: watch_puppetserver_mem.sh 20 > logfile.csv
+#
+# Example output: 
+#  time,mem
+#  10,2000
+#  20,2100
+#  30,2150
+
+
+delay=10
+if [ "$1" != "" ]
+then
+   delay=$1
+fi
+
+# Output PID
+echo $$ > watch_puppetserver_mem.pid
+
+# Dummy value
+origpid="firstpid"
+
+# What process to look for
+process_name="puppet-server"
```

Comment:
Yeah I noted during manual testing that it still worked with PE, and left it. But we could make the  regex `(puppet-server|pe-puppetserver)`?